tests/cli/test_suite.py

```diff
@@ -993,14 +997,12 @@ def test_suite_edit_datasource_and_batch_request_error(
 ):
     """This is really only testing click missing arguments"""
     context = empty_data_context_stats_enabled
-    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    monkeypatch.chdir(os.path.dirname(context.root_directory))  # noqa: PTH120
 
     expectation_suite_name: str = "test_suite_name"
 
     # noinspection PyUnusedLocal
-    suite: ExpectationSuite = context.add_expectation_suite(
-        expectation_suite_name=expectation_suite_name
```

Comment:
Do we not want to save the suite returned from add_expectation_suite in these 2 places?